Makefile

```diff
@@ -1,11 +1,14 @@
 RPM_PATH = "iguazio_yum"
 DEB_PATH = "iguazio_deb"
 BINARY_NAME = "igz-fuse"
-VERSION = $(IGUAZIO_VERSION:igz_%=%)
```

Comment:
Merge blooper, reset all changes make on Makefile and use the one on development.